src/graph/controller.py

```diff
@@ -43,7 +51,12 @@ def __init__(self):
         self._deployment_lock = asyncio.Lock()
 
     async def run(self):
-        # Create service if it doesn't already exist.
+        """
+        Spawn non-terminating tasks to watch changes to RampartGraph CRD objects and update
```

Comment:
It should CR objects not CRD objects, same to the line below.
I've added a comment for each of such appearances. But to make our lives easier I think we can delete all the CRDs if there is already a RampartGraph before it.

Reply:
I think its important to distinguish between Rampart graphs as a general, abstract concept and RampartGraph CR's as a specific kubernetes object. The CR makes it clear that its not a missing space or anything like that.